src/spikeinterface/widgets/unit_summary.py

```diff
@@ -17,19 +18,27 @@ class UnitSummaryWidget(BaseWidget):
     """
     Plot a unit summary.
 
-    If amplitudes are alreday computed they are displayed.
+    If amplitudes are alreday computed, they are displayed.
 
     Parameters
     ----------
-    sorting_analyzer : SortingAnalyzer
+    sorting_analyzer: SortingAnalyzer
```

Comment:
@florian6973 the space is needed for the docs and to comply with numpydocs. Can you re-add the space before the : to all arguments?